website/docs/dotnetsdk.md

```diff
@@ -0,0 +1,47 @@
+Selects a .NET Sdk
+
+```lua
+dotnetsdk "sdk"
+```
+
+[overview](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/overview)
+
+## parameters ##
+`sdk` is one of
+ * [web](https://learn.microsoft.com/en-us/aspnet/core/razor-pages/web-sdk?toc=%2Fdotnet%2Fnavigate%2Ftools-diagnostics%2Ftoc.json&bc=%2Fdotnet%2Fbreadcrumb%2Ftoc.json)
+ * [razor](https://learn.microsoft.com/en-us/aspnet/core/razor-pages/sdk?toc=%2Fdotnet%2Fnavigate%2Ftools-diagnostics%2Ftoc.json&bc=%2Fdotnet%2Fbreadcrumb%2Ftoc.json)
+ * [worker](https://learn.microsoft.com/en-us/dotnet/core/extensions/workers)
+ * [blazor](https://learn.microsoft.com/en-us/aspnet/core/blazor/)
+ * [windowsdesktop](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props-desktop?view=aspnetcore-8.0)
+ * [mstest](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-sdk)
+
+
+## mstest ##
+to make the MSTest sdk work you need to add the version to `global.json`:
```

Comment:
Would it make sense to generate that file too?

Reply:
yes i was thinking about that but how would i make it so you could specify the version? or do i just have a default version and tell people where they can change it! and code for writing the file as i am not familliar with the file api's of premake